src/components/queries/query1-page/query1-page.jsx

```diff
@@ -1,108 +1,109 @@
 import "./query1-page.css";
 import * as React from "react";
+import Plot from 'react-plotly.js';
 
-import Box from '@mui/material/Box';
+import { Box, FormGroup, FormControl, FormLabel, FormControlLabel,   
+        Select, MenuItem, Checkbox, Divider, Typography, Radio, RadioGroup} from '@mui/material';
 
-import FormGroup from '@mui/material/FormGroup';
-import Checkbox from '@mui/material/Checkbox';
 
-import Radio from '@mui/material/Radio';
-import RadioGroup from '@mui/material/RadioGroup';
-import FormControlLabel from '@mui/material/FormControlLabel';
-import FormControl from '@mui/material/FormControl';
-import FormLabel from '@mui/material/FormLabel';
+function Template() {
 
-import { Select, MenuItem } from '@mui/material';
+  const [season, setSeason] = React.useState('');
+  const [covidStatus, setCovidStatus] = React.useState('');
 
-function Template() {
+  const changeSeason = (event) => {
+    setSeason(event.target.value);
+  };
+  const changeCovidStatus = (event) => {
+    setCovidStatus(event.target.value);
+  };
 
   return (
     <div className="query-1-page">
-      <Box sx={{ flexGrow: 1,  height: 700}}>
+      <Box sx={{ flexGrow: 1,  height: 1000}}>
         <h1>Seasonal L.A. Crime Throughout COVID-19 Seasons</h1>
-        <hr class="section-divider"/>
-        <p class="subheading">An overview of crime trends across the four seasons (Spring, Summer, Fall, and Winter) in L.A.
-          (averaged from 2017 to 2022 with COVID-19 consideration)
```

Comment:
![image](https://user-images.githubusercontent.com/66442410/231643893-2c82f22e-fec1-4e25-ab48-053eca0c4035.png)

to be changed wrt to change in dataset.